### DISCUSSION TITLE
Pod not found and no content was found

### DISCUSSION THREAD

**Discussion by rrg92** [pods-framework/pods#7364] [Q&A]
Hi, after updating plugin and wordpress to latest versions (3.2.7 and 6.6.2, respectivelly), pods wont list.

In editor, when try select a template ,it shows "no content was found"

When try creating a new pod, it shows "No pod found".

some help?

**Comment by SupaDigital** [comment]
Having similar issues whereby templates no longer work.

**Reply by rrg92** [reply]
Hi, in my case, I found problem: https://github.com/pods-framework/pods/issues/7373

**Reply by SupaDigital** [reply]
Thanks. I managed to solve my issue just before you messaged actually. Turned out to be an issue related to the new access permissions whcih I FINALLY managed to figure out. Essentially, templates werent loading shortcodes. So I removed the php echo code around the shortcode, leaving just the shortcode, and it worked.
